src/platform/utils.py

```diff
@@ -0,0 +1,106 @@
+"""module to automatically wrap methods in autogenerated low-level code and re-expose them as high-level functions"""
+
+import inspect
+from urllib.parse import urljoin
+
+import do_sdk_platform
+from beartype import beartype
+from box import Box
+from deeporigin.auth import get_tokens
+from deeporigin.config import get_value
+from deeporigin.utils.core import _get_method
+
+
+@beartype
+def _get_api_client(*, api_name: str, configure: bool = True):
+    """return a configured client for the API we want to access
+
+    Args:
+        api_name (str): name of the API
+
+    Returns:
+        configured client
+    """
+
+    if configure:
+        configuration = do_sdk_platform.configuration.Configuration(
+            host=urljoin(get_value()["api_endpoint"], "/api"),
+            access_token=get_tokens()["access"],
+        )
+
+        client = do_sdk_platform.ApiClient(configuration=configuration)
+    else:
+        client = do_sdk_platform.ApiClient()
```

Comment:
if configure = False, we don't need it configured, we just need that object to inspect and dynamically build functions